src/defaultUserImage.ts

```diff
@@ -28,3 +33,29 @@ export const getDefaultUserImage = async (): Promise<ImageData> => {
         throw new Error('unsupported platform: ' + Platform.OS);
     }
 };
+
+export const copyImageToApp = async (image: ImageData, filename: string): Promise<string> => {
+    if (image.data != null) {
+        const path = await getPathForFile(filename);
+        if (!await RNFS.exists(path)) {
+            try {
+                await RNFS.writeFile(path, image.data, 'base64');
+            } catch (e) {
+                Debug.log('copyImageToApp', path, e);
+                throw e;
+            }
+        }
+        return `${FILE_PROTOCOL}${path}`;
+    } else if (image.localPath != null && isString(image.localPath)) {
+        return await copyToAppDataDir(getAbsolutePathFromLocalPath(image.localPath), filename);
+    } else {
+        throw Error(`copyImageToApp, unsupported image ${image}`);
+    }
+};
+
+export const createUserImage = (hash = generateUnsecureRandomHexString(15)): ImageData => {
```

Comment:
Why `15`? :)

Reply:
that was the minimum for `Identicon.js` 😄